composer.json

```diff
@@ -28,7 +28,7 @@
     "automattic/vipwpcs": "^2.3"
   },
   "scripts": {
-    "lint": "phpcs . --runtime-set testVersion 7.4-",
```

Comment:
`testVersion` needs to be specified here, the value is defined in `10up-Default`'s xml file and runtime values can [only be overridden by the command line](https://github.com/10up/phpcs-composer/pull/17#issuecomment-457922189).